packages/accounts-express/accounts_express_client_tests.js

```
@@ -0,0 +1,302 @@
+
+      const response = await Meteor.fetch(testUrl, {
+        ...customOptions,
+        headers: customHeaders
+      });
+      test.isTrue(response.ok);
+    } finally {
+      window.fetch = originalFetch;
+    }
+  });
+
+  Tinytest.addAsync('accounts-express - Meteor.fetch - works with session storage', async (test) => {
```

ESLint

[error] Replace `'accounts-express·-·Meteor.fetch·-·works·with·session·storage',·async·(test)` with `⏎····'accounts-express·-·Meteor.fetch·-·works·with·session·storage',⏎····async·test`

(prettier/prettier)